cmd/go-clang-generate/generate/astutil.go

```diff
@@ -150,6 +150,27 @@ func doZero() *ast.BasicLit {
 	}
 }
 
+func doSwitchStmt(tag ast.Expr) *ast.SwitchStmt {
+	return &ast.SwitchStmt{
+		Tag:  tag,
+		Body: &ast.BlockStmt{},
+	}
+}
+
+func doCaseClause(c []ast.Expr, b []ast.Stmt) *ast.CaseClause {
+	return &ast.CaseClause{
+		List: c,
+		Body: b,
+	}
+}
+
+func doStringLit(value string) *ast.BasicLit {
+	return &ast.BasicLit{
+		Kind:  token.STRING,
+		Value: "\"" + value + "\"",
```

Comment:
Do fmt.Sprintf("%q", value) instead since there might be characters in value that need escaping.